quantum/split_common/serial_backward_compatibility.h

```diff
@@ -0,0 +1,11 @@
+/* serial.h backward compatibility */
+
+// #ifndef SOFT_SERIAL_PIN
+//  #define SOFT_SERIAL_PIN D0
+// #endif
+
+#ifndef SERIAL_SLAVE_BUFFER_LENGTH
```

Comment:
Couldn't this be added to `serial.h`, rather than using a separate file?

Reply:
Do not worry, this `serial_backward_compatibility.h` will be deleted in the next phase.